backend/tests/api/test_feedback.py

```diff
@@ -41,68 +40,25 @@
                 "feedback_text": "",
             },
             200,
-        ),  # Feedback text is empty but feedback can still be submitted
+        ),
     ],
 )
-def test_feedback_submission(feedback_data: dict, expected_status: int) -> None:
-    headers = {
-        "accept": "application/json",
-        "Authorization": f"Bearer {API_SECRET_KEY}",
-    }
-    response = client.post("/feedback", json=feedback_data, headers=headers)
-    # Dump the response as JSON to validate structure
-
-    assert (
-        response.status_code == expected_status
-    )  # Ensure the response contains the "feedback" key
-
-    feedback_response = response.json()
-    assert (
-        "feedback" in feedback_response
-    )  # Ensure the response contains the "feedback" key
-
-
-def test_get_feedback_by_chat_id() -> None:
-    chat_id = str(uuid4())
-    headers = {
-        "accept": "application/json",
-        "Authorization": f"Bearer {API_SECRET_KEY}",
-    }
-
-    sample_feedback = {
-        "user_name": "Test User",
-        "chat_id": chat_id,
-        "like": True,
-        "feedback_text": "This is a test feedback comment.",
-    }
-
-    # Submit feedback
-    create_response = client.post(
-        "/feedback",
-        json=sample_feedback,
-        headers=headers,
+@patch.object(FeedbackService, "submit_feedback", new_callable=AsyncMock)
```

Comment:
Can we use monkeypatch instead since we are using pytest?

Reply:
Also, why do we need to mock this? We have a test db running during pytest and can use that.